src/Hosting/Hosting/src/Internal/HostingEnvironment.cs

```diff
@@ -1,9 +1,12 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.AspNetCore.Hosting;
+
+[DebuggerDisplay("ApplicationName = {ApplicationName}, EnvironmentName = {EnvironmentName}")]
```

Comment:
Should we include the `ContentRootPath` and `WebRootPath` too?

Reply:
They could be long and displayed as cut-off in VS. Showing a cut-off path would be unhelpful.

It's simple enough to look at the properties on the environment to see these values.